src/mlpack/methods/reinforcement_learning/policy/exp_decay_greedy_policy.hpp

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file exp_decaay_greedy_policy.hpp
```

Comment:
```suggestion
 * @file methods/reinforcement_learning/policy/exp_decay_greedy_policy.hpp
```
this should fix the doc build